content/doc/developer/handling-requests/json.adoc

```diff
@@ -0,0 +1,255 @@
+---
+title: Expose HTTP API with json content with Jenkins
+summary: How to write GET and POST HTTP API with Jenkins, and how to test it.
+layout: developersection
+references:
+- url: https://github.com/stapler/stapler/blob/master/docs/reference.adoc
+  title: Stapler URL Binding Reference
+- url: ../routing
+  title: How requests in Jenkins are routed
+- url: ../../testing
+  title: Testing Jenkins
+---
+
+== Expose HTTP API with json content with Jenkins
```

Comment:
This line should be removed as it duplicates the title with the `+ title:` directive.